src/tests/AccountCreation.test.js

```diff
@@ -0,0 +1,35 @@
+import '@testing-library/jest-dom/extend-expect'
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { postCheckingAccounts } from '../api';
+import {CheckingAccountCreation} from '../components/CheckingAccountCreation'
+
+//this test is not finished yet
+jest.mock('../api/index.js');
+
+test('Should create the checking account with the given info',async()=>{
+
+    const response = {
+      status:200
+    }
+
+    postCheckingAccounts.mockResolvedValue(response); 
+    
+    const form = render(<CheckingAccountCreation/>)
+
+    const name = form.getByTestId('account-name');
+    const deposit = form.getByTestId('initial-deposit');
+    const submit = form.getByTestId('submit'); 
+
+    fireEvent.change(name,{target:{value:'testing'}}); 
+    fireEvent.change(deposit,{target:{value:'2500'}}); 
+    fireEvent.click(submit);  
+
+    await waitFor(()=>{
+      expect(form.getByTestId('success')).toBeInTheDocument(); 
+    })
+    
```

Comment:
You could also check the Checking Accounts list to validate that the created account is actually there.

Reply:
I don't know how to do that yet, but I will look on the internet